docs/pages/07_service.adoc

```diff
@@ -168,25 +168,83 @@ transfer PersonTransfer {
 
 The second example defines the AddressTransfer transfer object with its fields, and each PersonTransfer instance must have an AddressTransfer.
 
-// read-only fields?
-
-=== Default value
-
 Optionally, a <default> value can be specified as an expression for transfer object fields. The <default> value expression is evaluated when a new instance of the transfer object is created, and the field is set according to the default value expression.
 
 *Example:*
 
 ----
 transfer PersonTransfer {
-    field required String firstName ;
-    field required String lastName ;
+    field required String firstName = "John";
+    field required String lastName = "Doe";
     field String midName = "" ;
 }
 ----
 
-The example above complements the previous PersonTransfer example by adding a default value to the "midName" field. The default value of the `midName` is an empty string.
+The example above complements the previous PersonTransfer example by adding a default value to the `firstName`, `lastName` and `midName` fields.
+
+// read-only fields?
+
+=== Mapping configuration
+
+Mapping configuration is a declarative description of how the fields of a transfer object read and/or write specific fields of the entity mapped by the transfer object. Mapping configuration can only be used for mapped transfer objects.
+
+Use the `reads` or `maps` keywords in the field definitions to define the mapping configuration.
+
+*Syntax:*
+
+----
+field [required] <primitive> <name> [= <default>] [(reads|maps) <expression>];
+----
+
+where <primitive> is the name of a domain model primitive, or
+
+----
+field [required] <transfer>[[]] <name> [= <default>]  [(reads|maps) <expression>];
+----
+
+If the `reads` keyword is used, the value of the transfer field will be set to the value of the expression specified after `reads`. 
+
+*Example:*
+
+----
+entity SalesPerson {
+    field String firstName ;
+    field String lastName ;
+}
+
+transfer SalesPersonTransfer(SalesPerson salesperson) {
+    field String fullName reads salesperson.firstName + " " + salesperson.lastName ;
+}
+----
+
+In the example above, transfer field value is set by concatenating the `firstName` and `lastName` fields of SalesPerson mapped entity with a space in the middle. Note the `salesperson` reference in the expression which refers to the mapping field.
 
-The default value can be used for the mapping configuration in mapped transfer objects.
+The `reads` expressions are used whenever a transfer object or a collection of transfer objects are requested in a function, see <<Data function>> section.
+
+In the second example, we use the `maps` keyword, which in addition to setting the value of the transfer field, writes the value of the transfer field to the entity field when the transfer object is saved.
```

Comment:
The "transfer object is saved" expression is not correct exactly.
My suggestions, but they aren't also very good, because there have not been defined the saving mechanism.
... when the mapped entity instance is updated with the transfer object instance.
... when called the tranfer object's update mechanism.

Reply:
done